acquisitions/src/main/resources/thunderjet/mod-orders/features/close-order-when-fully-paid-and-received.feature

```diff
@@ -1,84 +1,115 @@
+    * def v = call openOrder { orderId: "#(orderId)" }
 
-  Scenario: Create order line
-    * def v = callonce createOrderLine { id: #(poLineId), orderId: #(orderId), fundId: #(globalFundId) }
-
-  Scenario: Open order
-    * def v = callonce openOrder { orderId: "#(orderId)" }
-
-  Scenario: Get poLine and update payment and receipt status
+    # 2 Get poLine and update payment and receipt status to resolution
     Given path 'orders/order-lines', poLineId
     When method GET
     Then status 200
-
-    * def poLineResponse = $
+    * def poLineResponse = response
     * set poLineResponse.paymentStatus = 'Fully Paid'
     * set poLineResponse.receiptStatus = 'Fully Received'
 
+    # 3. PUT updated PoLine
     Given path 'orders/order-lines', poLineId
     And request poLineResponse
     When method PUT
     Then status 204
 
-  Scenario: Check that order closed
+    # 4. Check that order closed
     Given path 'orders/composite-orders', orderId
-    When method GET
     And retry until response.workflowStatus == 'Closed'
-    Then status 200
-
-    Given path 'orders/composite-orders', orderId
     When method GET
     Then status 200
-    And match $.workflowStatus == 'Closed'
+    And match response.workflowStatus == 'Closed'
 
-  Scenario: Check the encumbrance was released
+    # 5. Check the encumbrance was released
     Given path 'finance/transactions'
     And param query = 'transactionType==Encumbrance and encumbrance.sourcePurchaseOrderId==' + orderId + ' and encumbrance.status==Released'
     When method GET
     Then status 200
-    And match $.totalRecords == 1
+    And match response.totalRecords == 1
 
-  Scenario: delete poline
+    # 6. Delete order and PoLine
     Given path 'orders/order-lines', poLineId
     When method DELETE
     Then status 204
 
-  Scenario: delete composite orders
     Given path 'orders/composite-orders', orderId
     When method DELETE
     Then status 204
+
+
+  Scenario: Closed order should not be reopened when it has at least one resolution status
+    # 1. Create order, order line and open order
+    * def orderId = call uuid
+    * def poLineId = call uuid
+    * def v = call createOrder { id: #(orderId) }
+    * def v = call createOrderLine { id: #(poLineId), orderId: #(orderId), fundId: #(globalFundId) }
+    * def v = call openOrder { orderId: "#(orderId)" }
+
+    # 2. Close order
+    Given path 'orders/composite-orders', orderId
+    When method GET
+    Then status 200
+    * def order = response
+    * set order.workflowStatus = 'Closed'
+
+    Given path 'orders/composite-orders', orderId
+    And request order
+    When method PUT
+    Then status 204
+
+    # 3 Get poLine and update payment to resolution
+    Given path 'orders/order-lines', poLineId
+    When method GET
+    Then status 200
+    * def poLineResponse = response
+    * set poLineResponse.paymentStatus = 'Fully Paid'
+
+    # 3. PUT updated PoLine
```

Comment:
as a option we can remove this. it is already clear to the purpose of GET and PUT from comment
https://github.com/folio-org/folio-integration-tests/pull/1537/files#diff-d3dfe159a64228566e27f962317399b9729f216c03054f2114cfa6654d4bb276R89